src/main.cpp

```diff
@@ -1,8 +1,23 @@
 #include <iostream>
+#include "../include/InventoryManager.hpp"
 
-int main(void){
+int main(int argc, char**argv){
 
-	std::cout << "Hi guys!" << std::endl;
+	std::string arg;
+	bool command_line = false;
 
+	if (argc == 2) {
```

Comment:
For consistency purposes, it'd be nice if we all put ```{}```s on their own line.

However, I was looking over [ActiveInventory.cpp](https://github.com/QT-Pies/inventory-project/blob/main/src/ActiveInventory.cpp) again, and it has a few conditionals that have {'s on the same line, so it's up to you on if you want to keep them like this or match the style guide; it seems like the majority of the team likes to put them on the same line, so perhaps I'll ask in the Discord again about everyone's opinion on the style guide.

Feel free to mark this as resolved, you don't have to change these unless you feel like it.